container/cli.py

```diff
@@ -243,5 +243,5 @@ def commandline():
         if args.debug:
             logger.exception(e)
         else:
-            logger.error(e)
+            logger.error('Execution failed with {}'.format(type(e)))
```

Comment:
@yanchenko, not sure we always want to see `type(e)`. What we want is a string representation of `e`. Maybe this should check if `str(e)` returns anything, and if not, then return `type(e)`?  I'm concerned that standard exceptions won't result in a text message. Have you tested with a more typical exception?